samples/subsys/tracing/README.rst

```diff
@@ -0,0 +1,87 @@
+.. _tracing_sample:
+
+Send Tracing Formatted Packet To The Host With Supported Backends
+#################################################################
+
+Overview
+********
+
+This application can be used to demonstrate the tracing feature. The tracing
+formatted packet will be sent to the host with the currently supported tracing
+backend under tracing generic infrastructure.
+
+Requirements
+************
+
+Depends of the boards which you are using, choose one of .conf files for use tracing subsys.
+
+Usage for UART Tracing Backend
+******************************
+
+Build a UART-tracing image with:
+
+.. code-block:: console
+
+	cmake -DBOARD=mps2_an521 -DCONF_FILE=prj_uart.conf ..
+
+or:
+
+.. code-block:: console
+
+	cmake -DBOARD=mps2_an521 -DCONF_FILE=prj_uart_ctf.conf ..
+
+- NOTE: You may need to set 'zephyr,tracing-uart' property under the chosen
```

Comment:
You can use the `.. note::` directive here. The doc currently fails to build due to this line anyways (see https://github.com/zephyrproject-rtos/zephyr/actions/runs/4367897219/jobs/7654145083)

Also please use backquotes around "zephyr,tracing-uart" instead of simple quotes.